plugins/vst_base/CMakeLists.txt

```diff
@@ -39,17 +39,17 @@ IF(LMMS_HOST_X86_64)
 ENDIF(LMMS_HOST_X86_64)
 
 ADD_CUSTOM_COMMAND(
-		SOURCE "${CMAKE_CURRENT_SOURCE_DIR}/RemoteVstPlugin.cpp"
```

Comment:
This entire section may be worth breaking out into its own block under `IF(LMMS_BUILD_APPLE)` (or perhaps IF-clang).  Also, a description as to why these changes are needed would go a long way to understanding why these are needed.